Code/Editor/Plugins/ProjectSettingsTool/ProjectSettingsToolWindow.cpp

```diff
@@ -33,12 +33,24 @@
 #include <QScrollBar>
 #include <QTimer>
 
-// The object name in json for android
-const static char* androidSettings = "android_settings";
-static bool g_serializeRegistered = false;
-
 namespace ProjectSettingsTool
 {
+    namespace
+    {
+        const char* const IosSettingsPListPaths[] = {
+                    "Resources/Platform/iOS/Info.plist",
```

Comment:
Nitpick: There looks to be an extra tab spacing here

Reply:
Good spot. I'll fix it.